CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ## 0.7.0
 
 ### Vision
-- Changed country-based telemetry synchronization
+- Improved lane detection
```

Comment:
@chezzdev @IodaMikeMapbox, I'm sorry, didn't remember if we had an agreement to use strict set of verbs as a first word in a CHANGELOG's row?

If yes, everything looks good, otherwise - we can update it with `Changed X implementation to improve Y...` or `Added support of X...`.

Reply:
We didn't have a commitment to use it, we use it as a general guideline.